.ci/util.sh

```diff
@@ -25,6 +25,10 @@ function getCheckstylePomVersion {
   getMavenProperty project.version
 }
 
+function getCheckstylePomVersionWithoutSnapshot {
+  getCheckstylePomVersion | sed "s/-SNAPSHOT//"
```

Comment:
please create function in this file
to keep implementation of 
```
xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
                             -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//"
```
one day it will be convenient to use, as we not always use "mvn" in builds.

Such function will be unused for now, it is ok.

Reply:
How shall we name it? `getCheckstylePomVersionWithoutSnapshotWithXmlstarlet`?

Reply:
I ok, I have no better name.

Reply:
Done.